src/components/views/voip/VideoView.js

```diff
@@ -50,7 +50,10 @@ module.exports = React.createClass({
     },
 
     getRemoteAudioElement: function() {
-        return this.refs.remoteAudio;
+        // this needs to be somewhere at the top of the DOM which
+        // always exists to avoid audio interruptions.
+        // Might as well just use DOM.
+        return document.getElementById("remoteAudio");
```

Comment:
Actually I think it may be worse than that currently - it won't NPE it all, it'll just not play any audio (JS SDK has null guards on).